tests/clone_test.go

```diff
@@ -840,3 +852,31 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 		})
 	})
 })
+
+func startCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (*virtv1.VirtualMachine, error) {
+	patch, err := patch.New(patch.WithReplace("/spec/runStrategy", virtv1.RunStrategyAlways)).GeneratePayload()
+	if err != nil {
+		return nil, err
+	}
+
+	updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, v1.PatchOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	return updatedVM, err
+}
+
+func stopCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (*virtv1.VirtualMachine, error) {
+	patch, err := patch.New(patch.WithReplace("/spec/runStrategy", virtv1.RunStrategyHalted)).GeneratePayload()
+	if err != nil {
+		return nil, err
+	}
+
+	updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, v1.PatchOptions{})
```

Comment:
- You could return directly here.
- You could avoid passing virt-client (as it is a singleton).